src/FormGenerator.js

```diff
@@ -0,0 +1,95 @@
+//Form creation tool
+module.exports.generateForm = function(input){
+  //String variable to hold final form render
+  var formStringHTML = '';
+
+  //For each data in input array
+  input.forEach(function(input){
+    //Switch to determine & handle specific input type
+    switch(input.type){
+      //LABEL
+      case 'label':
+        formStringHTML += '<label';
+        //Add 'for' attribute if ID is avail
+        input.id ? formStringHTML += ' for="' + input.id + '"></label>' : formStringHTML += '></label>';
+
+        break;
+
+      //TEXT
+      case 'text':
+        //BEGIN Input
+        formStringHTML += '<input type="text"';
+
+        //ADD Attributes if present
+        input.name ? formStringHTML += ' name="' + input.name + '"' : formStringHTML += '';
```

Comment:
These ternary exps could be simplified a little bit: `formStringHtml += input.id ? ' id="' + input.id + '"' : '';`

Style change to consider--Style guide may also dictate template strings instead of concatenation: 
```
formStringHTML += input.name ? ` name="${input.name}"` : ``;
```